lib/Interpreter/ValuePrinter.cpp

```diff
@@ -398,71 +385,59 @@ static std::string printFunctionValue(const Value &V, const void *ptr, clang::Qu
   return functionString;
 }
 
-static std::string printUnpackedClingValue(const Value &V) {
-  std::stringstream strm;
+static std::string printAddress(const void* Ptr, const char Prfx = 0) {
+  if (!Ptr)
+    return kNullPtrStr;
+
+  llvm::SmallString<256> Buf;
+  llvm::raw_svector_ostream Strm(Buf);
+  if (Prfx) Strm << Prfx;
```

Comment:
I'd prefer to break the if statements. Could you more the then statement on a new line. Makes it easier for debugging with gdb.